src/Applications/LDAS_App/process_rst.csh

```diff
@@ -79,55 +79,34 @@ if ( -e /etc/os-release ) then
 else
   module load other/nco-4.6.8-gcc-5.3-sp3 
 endif
-#setenv OMPI_MCA_shmem_mmap_enable_nfs_warning 0
-#setenv MKL_CBWR SSE4_2 # ensure zero-diff across archs
-#setenv MV2_ON_DEMAND_THRESHOLD 8192 # MVAPICH2
+
 setenv LAIFILE `find ${BCSDIR}/lai_clim*`
 setenv LD_LIBRARY_PATH ${LD_LIBRARY_PATH}:${BASEDIR}/Linux/lib
 
 limit stacksize unlimited
  
-$INSTDIR/bin/esma_mpirun -np 56 bin/mk_GEOSldasRestarts.x -a ${SPONSORID} -b ${BCSDIR} -t ${TILFILE} -m ${MODEL} -s ${SURFLAY} -j Y
+$INSTDIR/bin/esma_mpirun -np 56 bin/mk_GEOSldasRestarts -a ${SPONSORID} -b ${BCSDIR} -t ${TILFILE} -m ${MODEL} -s ${SURFLAY} -j Y
```

Comment:
Hard-coding 56 processors isn't exactly efficient on skylake nodes, which have 40 cores per node.  Can we specify the number of nodes instead, or do something else that relies less on hard-coded numbers?  
The same comment applies to a couple more mpirun calls in this script.